src/Microsoft.Azure.WebJobs.Host.Storage/IAzureStorageProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Azure.Storage.Blobs;
+
+namespace Microsoft.Azure.WebJobs.Host.Storage
+{
+    /// <summary>
+    /// Interface to retrieve BlobServiceClient objects
+    /// </summary>
+    public interface IAzureStorageProvider
+    {
+        bool ConnectionExists(string connection);
```

Comment:
This doesn't seem like it belongs on this interface. We have existing Configuration extensions and it looks like your impl. is just delegating to those.

Reply:
Purpose of this method is to determine if the backing IConfiguration for this object has the given connection. Yes, it uses the configuration extensions, but implementations of IAzureStorageProvider may maintain different IConfigurations (i.e. Host versus default AzureStorageProvider registered in Host.Storage).